charts/kube-prometheus-stack/README.md

```diff
@@ -83,6 +83,13 @@ _See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documen
 
 A major chart version change (like v1.2.3 -> v2.0.0) indicates that there is an incompatible breaking change needing manual actions.
 
+### From 26.x to 27.x
```

Comment:
Can you add a note that this adds the runbook url back?

Reply:
Didn't put it in because no breaking change... But yes :)